src/pathfinding_igvc/test/path-finder-utils-test.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Created By: Min Gyo Kim
+ * Created On: May 14th 2018
+ * Description: Unit tests for path finder utils class
+ */
```

Comment:
this file should probably be called `pathfinder-test-utils.cpp`, `path-finder-utils-test` makes it sound like we're testing the utils

Reply:
This file is actually testing `PathFinderUtils`

Reply:
Ah, that makes sense, all good.